tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -126,6 +126,24 @@ EOF
 	exit "${return_code}"
 }
 
+get_kernel_modules_dir() {
+	local kernel_version="${1:-}"
+	local kernel_kata_config_version="${2:-}"
+	local kernel_name"=${3:-}"
+	[ -z "${kernel_version}" ] && die "kernel version is a required argument"
+	[ -z "${kernel_kata_config_version}" ] && die "kernel kata config version is a required argument"
+	[ -z "${kernel_name}" ] && die "kernel name is a required argument"
+
+	local numeric_final_version=${kernel_version#v}
```

Comment:
This value isn't being used? Should it replace `${kernel_version#v}` in the echo, or be removed?

Reply:
Fixes, thanks!